Reorg DOT page (#3841)
w3f/polkadot-wiki: docs/learn/learn-DOT.md

DOT is the native token of the Polkadot network in a similar way that BTC is the native token of
Bitcoin or Ether is the native token of the Ethereum blockchain.

### The Planck Unit

The smallest unit of account in a Substrate network (Polkadot, Kusama, etc.) is the Planck (a
reference to [Planck Length](https://en.wikipedia.org/wiki/Planck_length), the smallest possible
distance in the physical Universe). You can compare the Planck to Satoshis or Wei, while the DOT is
like a BTC or an ETH. Kusama tokens (KSM) are equal to 10<sup>12</sup> Planck, and Polkadot mainnet

> DOT is equal to 10<sup>10</sup> Planck.

Reviewer: Consider replacing the entire paragraph with this text. Could not make suggestions to the lines that were not part of this PR's edits

"The smallest unit for the account balance on Substrate based blockchains (Polkadot, Kusama, etc.) is 
Planck (a reference to [Planck Length](https://en.wikipedia.org/wiki/Planck_length), the smallest 
possible distance in the physical Universe). You can compare DOT's Planck to BTC's Satoshi or ETH's 
Gwei. Polkadot's native token DOT equals to 10<sup>10</sup> Planck and Kusama's native token KSM
equals to 10<sup>12</sup> Planck."

Author: Thanks, replaced as suggested

### Polkadot

| Unit            | Decimal Places | Conversion to Planck   | Conversion to DOT |
| --------------- | -------------- | ---------------------- | ----------------- |
| Planck          | 0              | 1 Planck               | 0.0000000001 DOT  |
| Microdot (uDOT) | 4              | 10<sup>4</sup> Planck  | 0.0000010000 DOT  |
| Millidot (mDOT) | 7              | 10<sup>7</sup> Planck  | 0.0010000000 DOT  |
| Dot (DOT)       | 10             | 10<sup>10</sup> Planck | 1.0000000000 DOT  |
| Million (MDOT)  | 16             | 10<sup>16</sup> Planck | 1,000,000.00 DOT  |

:::note DOT was redenominated at block #1_248_328

> Previously, DOT was denominated as equal to 10<sup>12</sup> Planck, just like Kusama. This
> denomination is deprecated, and, if necessary, referred to as "DOT (old)". DOT supply was increased
> by 100x and as a consequence now 1 DOT = 10<sup>10</sup> Planck. See
> [Redenomination of DOT](../general/redenomination.md) for more details.

Reviewer: ```suggestion
DOT was originally equal to 10<sup>12</sup> Planck just like Kusama (which is referred to as "DOT (old)"), but went through a 
process of [redenomination](../general/redenomination.md) which increased DOT's supply
by 100x. As a consequence, 1 DOT now equals to 10<sup>10</sup> Planck. 
```

   approximately 14’400 blocks per day, and you can see what the latest block is shown on the
   [Explorer](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frpc.polkadot.io#/explorer) page.

## Obtaining Testnet Tokens

DOT are required to make transactions on the Polkadot network. Testnet tokens exist on copies of the
Polkadot network (or its parachains) and do not have any value besides allowing you to experiment

> with the network.

Reviewer: ```suggestion
DOT are required to make transactions on the Polkadot network. Tokens on Polkadot's Testnets like Westend and Rococo do not have any value besides allowing you to experiment with the features on the network.
```

### Getting Tokens on the Westend Testnet

> The current Polkadot testnet is called

Reviewer: ```suggestion
Polkadot's testnet is called
```

[Westend](../maintain/maintain-networks.md#westend-test-network) and you can obtain its native

> tokens by posting `!drip <WESTEND_ADDRESS>` in the Matrix chatroom

Reviewer: ```suggestion
tokens (called Westies!) by posting `!drip <WESTEND_ADDRESS>` in the Matrix chatroom
```

[#westend_faucet:matrix.org](https://matrix.to/#/#westend_faucet:matrix.org). Your account will be
credited with 1 WND by default. You can also specify to get more tokens by
`!drip <WESTEND_ADDRESS> X`, where X is the number of tokens.

> You can make your WNDs (testnet DOT) by [becoming a validator](learn-validator.md). Watch the video

Reviewer: ```suggestion
You can also earn WNDs as rewards by [becoming a validator](learn-validator.md) on Westend network. Watch the video
```
